styles.css

```diff
@@ -36,22 +36,21 @@ a {
     color: #ececec;
 }
 
+h1 {
+    font-size: 2em;
+}
+
 /*Navigation*/
-#nav_bar {
+nav {
     position: fixed;
     top: 0;
     width: 100%;
     height: 4rem;
     background-color: #385170;
     z-index: 2;
-    /*display: flex;
-    flex-wrap: wrap;
-    flex-grow: 1;
-    justify-content: space-between;
-    align-items: center;*/

```

Comment:
Don't leave commented out code in your code base! As long as you've committed the original to git, just delete it and commit it again and you can always get it back out of git.

Reply:
I'd actually totally forgotten about that! Thanks for pointing that out!